internal/cmd/stack/stack.go

```diff
@@ -35,13 +35,39 @@ func Command() *cli.Command {
 				Usage:    "Discard an unconfirmed tracked run",
 				Flags: []cli.Flag{
 					flagStackID,
-					flagRun,
+					flagRequiredRun,
 					flagTail,
 				},
 				Action:    runDiscard(),
 				Before:    authenticated.Ensure,
 				ArgsUsage: cmd.EmptyArgsUsage,
 			},
+			{
+				Category: "Run management",
+				Name:     "approve",
+				Usage:    "Approves a run or task. If no run is specified, the approval will be added to the current stack blocker.",
```

Comment:
`Approves a run or task. If no run is specified, the approval will be added to the current stack blocker.` -> `Approves a run or task. If no run is specified, the current stack blocker will be approved.`